homeassistant/components/kat_bulgaria/binary_sensor.py

```diff
@@ -0,0 +1,64 @@
+"""Binary sensor platform."""
+
+from datetime import timedelta
+import logging
+
+from kat_bulgaria.obligations import KatApi, KatApiResponse
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .common import generate_entity_name
+from .const import CONF_DRIVING_LICENSE, CONF_PERSON_EGN, CONF_PERSON_NAME, DOMAIN
+
+SCAN_INTERVAL = timedelta(minutes=20)
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the platform from config_entry."""
+
+    person_name: str = str(entry.data[CONF_PERSON_NAME])
```

Comment:
We don't need to copy to string here. These items are already strings.

Reply:
I know but I got some type error here, so I used this to make it know what type it is.